### PR TITLE
Change installation YAML to point to gcr.io

### PR DESCRIPTION
**Pull request by kflynn**
We're seeing problems with at least some customers pulling from docker.io from EKS clusters.

### CODE REVIEW

File: docs/yaml/ambassador/ambassador-knative.yaml [hunk 1]
```diff
@@ -1402,7 +1402,7 @@ spec:
       serviceAccountName: ambassador
       containers:
       - name: ambassador
-        image: docker.io/datawire/ambassador:$version$
+        image: gcr.io/ambassador:$version$
```
Comment (cakuros) [inline docs/yaml/ambassador/ambassador-knative.yaml:1405]: should be `gcr.io/datawire/ambassador`?

File: docs/yaml/ambassador/ambassador-knative.yaml.m4 [hunk 2]
```diff
@@ -90,7 +90,7 @@ spec:
       serviceAccountName: ambassador
       containers:
       - name: ambassador
-        image: docker.io/datawire/ambassador:$version$
+        image: gcr.io/ambassador:$version$
```
Comment (cakuros) [inline docs/yaml/ambassador/ambassador-knative.yaml.m4:93]: gcr.io/datawire/ambassador

File: docs/yaml/ambassador/ambassador-rbac-prometheus.yaml [hunk 3]
```diff
@@ -1419,7 +1419,7 @@ spec:
             path: mapping-config.yaml
       containers:
       - name: ambassador
-        image: docker.io/datawire/ambassador:$version$
+        image: gcr.io/ambassador:$version$
```
Comment (cakuros) [inline docs/yaml/ambassador/ambassador-rbac-prometheus.yaml:1422]: gcr.io/datawire/ambassador

File: docs/yaml/ambassador/ambassador-rbac-prometheus.yaml.m4 [hunk 4]
```diff
@@ -107,7 +107,7 @@ spec:
             path: mapping-config.yaml
       containers:
       - name: ambassador
-        image: docker.io/datawire/ambassador:$version$
+        image: gcr.io/ambassador:$version$
```
Comment (cakuros) [inline docs/yaml/ambassador/ambassador-rbac-prometheus.yaml.m4:110]: gcr.io/datawire/ambassador

File: docs/yaml/ambassador/ambassador-knative.yaml (hunk 1 above)
Comment (kflynn) [reply docs/yaml/ambassador/ambassador-knative.yaml:1405]: Oh, nice catch.

**Review by cakuros — APPROVED** [review]
LGTM

**Comment by kflynn** [comment]
This was for Game Day. Whew. :wink:

**Pull request closed without merging**